test/e2e/daemon_set.go

```diff
@@ -522,7 +522,7 @@ func checkDaemonPodOnNodes(f *framework.Framework, selector map[string]string, n
 
 		nodesToPodCount := make(map[string]int)
 		for _, pod := range pods {
```

Comment:
This helper should filter out pods not owned by the tested daemon set and the following zero in IsPodAvailable should be ds.Spec.MinReadySeconds.